pkgs/by-name/st/sturmflut/package.nix

```
@@ -0,0 +1,42 @@
+{ stdenv
+, lib
+, pkg-config
+, imagemagick
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation {
+  pname = "sturmflut";
+  version = "git";
+
+  src = fetchFromGitHub {
+    owner = "TobleMiner";
+    repo = "sturmflut";
+    rev = "0e3092ab6db23d2529b8ddc95e5d5e2c3ae8fc9d";
+    hash = "sha256-amNkCDdfG1AqfQ5RCT4941uOtjQRSFt/opzE8yIaftc=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    imagemagick
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin
+    cp sturmflut $out/bin
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "A FAST (80+ Gbit/s) pixelflut client with full IPv6 and animation support";
```

The preference is to avoid use of indeterminate articles at the beginning of the description and use proper capitalisation, unless however `FAST` is indeed an acronym:
```suggestion
    description = "Fast (80+ Gbit/s) pixelflut client with full IPv6 and animation support";
```

---

Sounds good. I changed FAST to fast earlier today and was wondering about whether to drop the indeterminate article and decided against it. Will change that now.